dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ConjureBodySerDe.java

```diff
@@ -78,13 +78,7 @@
     @Override
     public Deserializer<Void> emptyBodyDeserializer() {
         return input -> {
-            try (Response response = input) {
-                if (response.body().read() != -1) {
-                    throw new SafeRuntimeException("Expected empty response body");
-                }
-            } catch (IOException e) {
-                throw new SafeRuntimeException("Failed to read from response body", e);
-            }
+            input.close();
```

Comment:
Let's leave a comment describing that `void`->`type` is allowed. Is there a quick test we can add?

Reply:
👍 I'll add a comment, I already added tests :)